File: Python3/vetor_crescente.py
"""Problema: Crie um algoritmo lendo um vetor A de 15 elementos do tipo inteiro.
 Mostre esses elementos do vetor A. Reordene os elementos em ordem crescente,
 atualize o vetor A com os elementos na ordem crescente e mostre novamente o vetor A."""
array = [9,3,7,6,21,50,45,36,2,8,7,9,42,32,18] 
aux = 0
for j in range(0,len(array)):
    print("Array[%d] = %d"%(j,array[j]))

for i in range(0,len(array)):
    for j in range(i+1,len(array)):
        if (array[i] > array[j]):
            aux = array[i]
            array[i] = array[j]
            array[j] = aux
            
print("Valores Ordenados")
for j in range(0,14):
    print("Array[%d] = %d"%(j,array[j]))

File: java/vetor_crescente.java
class vetor_crescente {
   public static void main(String args[]) {
      int A[] = { 9, 3, 7, 6, 21, 50, 45, 36, 2, 8, 7, 9, 42, 32, 18 };
      int Aux;
      String mensagem = "Valores Originais\n";
      for (int j = 0; j <= 14; j++)
         mensagem = mensagem + "A[" + j + "] =" + A[j] + " - ";

      for (int j = 0; j <= 13; j++)
         for (int i = 0; i <= 13; i++)
            if (A[i] > A[i + 1]) {
               Aux = A[i];
               A[i] = A[i + 1];
               A[i + 1] = Aux;
            }
      ;
      mensagem = mensagem + "\n\nValores Ordenados\n";
      for (int j = 0; j <= 14; j++)
         mensagem = mensagem + "A[" + j + "] =" + A[j] + " - ";
      System.out.println(mensagem);

   }
}
